AutomatedTesting/Gem/AssetProcessorGemConfig.setreg

```diff
@@ -17,6 +17,8 @@
                 "Exclude Restricted AssetProcessorTestAssets": {
                     "pattern": "(^|.+/)asset_processor_tests/restricted(/.+)$"
                 }
+                "Exclude Scene_Tests": {
```

Comment:
what is this excluding the tests from? the fast smoke test?

Reply:
It's making sure the assets this folder aren't scanned and processed by AP while building, same conditions as the current fbx_tests and asset_processor_tests.